blog/_posts/2015-05-04-release-notes-4.1-RC1.md

```diff
@@ -0,0 +1,85 @@
+
+### Expression evaluator
+
+This is by far the biggest feature in this release and also the most exciting one. Whenever you run an application in the debugger and you hit a breakpoint, you can evaluate Scala code in the context of your breakpoint. This is similar to what the worksheet or the REPL offers, with the difference that everything is evaluated while your application is suspended in debug mode and therefore can take as input the state of your program.
+
+For more information see the [documentation page][ee] of the expression evaluator.
+
+### Hot Code Replacement
+
+Beside of the big expression evaluator, some other nice little improvements went into the debugger. The most important to mention is Hot Code Replacement, a feature of the JVM, that allows a debugger to reload code inside of a running application. Unfortunately this only works for local code, i.e. no definitions that change the structure of your application. This means that you can replace the content of methods, recompile your code, and the debugger will reload the new code into your running application. You can't change any method signatures, nor add new methods or classes - if you want to do that you still have to restart your application. For more information about how to use this feature, see the [HCR docs][hcr].
+
+### New schema of compiling dependent projects
+
+TODO
+
+### Remove old wizards
+
+In 4.0, a [new wizard][new-wizard] has been introduced whose goal it was to be a simpler alternative to the old wizard. In the last months no bugs that couldn't be fixed were found in this new wizard, therefore we now completely replaced the old wizard with the new one.
+
+## Stats
+
+As for every release, we'll take a look at some statistics from our bug database.
+
+- 147 tickets closed
+- 71 fixed tickets (the difference being due to invalid or duplicate tickets)
+
+For this release, we went through a cleanup of our ticket database. In doing so a lot of outdated ticket got closed, therefore the top ticket closer for this release is Mr. Unassigned. Overall, the stats are a bit off this time because a lot of time was spent in implementing new features that had no corresponding ticket in the database.
+
+<pre>
+Tickets fixed:
+    41	(unassigned)
+    18	Simon Schäfer
+    8	Matthias Langer
+    2	Iulian Dragos
+    1	Jerzy Müller
+    1	Michał Pociecha
+</pre>
+
+## Thanks
+
+For this release, old contributors moved away, but new ones arrived. Beside from all the work VirtusLab did, special thanks goes to [Matthias Langer][mlangc] who made a great deal in improving this release.
+
+<pre>
+Commits:
+    60  Jerzy Müller
+    57  Matthias Langer
+    44  Simon Schäfer
+    12  Simon Schaefer
+    11  Iulian Dragos
+    11  Piotr Kukiełka
+     8  Michał Pociecha
+     7  Luc Bourlier
+     6  Krzysztof Romanowski
+     4  Brook Heisler
+     3  François Garillot
+     2  Aleksander Guryanov
+     2  Jason Zaugg
+     2  mpociecha
```

Comment:
I think it's ok to merge manually the count of Michał's commits - 10 Michał Pociecha